src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheckTest.java

```diff
@@ -98,4 +100,21 @@ public void shouldNotLogLongLinks() throws Exception {
         verify(checkConfig, getPath("InputLineLengthLongLink.java"), expected);
     }
 
+    @Test
+    public void countUnicodePointsOnce() throws Exception {
+        final DefaultConfiguration checkConfig =
+                createModuleConfig(LineLengthCheck.class);
+        checkConfig.addAttribute("max", "100");
+        // we need to set charset to let test pass on windows where default charset is not UTF-8
+        System.setProperty("file.encoding", StandardCharsets.UTF_8.name());
```

Comment:
In this case other tests will be affected.
Please see https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyCatchBlockCheckTest.java#L92

Reply:
I would recommend something like https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java#L515